src/Microsoft.ML.FastTree/FastTreeRanking.cs

```diff
@@ -405,6 +414,9 @@ protected override string GetTestGraphHeader()
             return headerBuilder.ToString();
         }
 
+        protected override RankingPredictionTransformer<FastTreeRankingPredictor> MakeTransformer(FastTreeRankingPredictor model, ISchema trainSchema)
+        => new RankingPredictionTransformer<FastTreeRankingPredictor>(Host, model, trainSchema, FeatureColumn.Name);
```

Comment:
>FeatureColumn.Name); [](start = 96, length = 20)

should add the GroupID to the base constructor

Reply:
GroupID? why?

---
In reply to: [216093781](https://github.com/dotnet/machinelearning/pull/855#discussion_r216093781) [](ancestors = 216093781)